docs/en/stack/ml/df-analytics/ml-trained-models.asciidoc

```diff
@@ -0,0 +1,28 @@
+[role="xpack"]
+[[ml-trained-models]]
+= Trained models
+
+experimental::[]
+
+When you use a {dfanalytics-job} to perform {classification} or {reganalysis},
+it creates a {ml} model that is trained and tested against a labelled data set.
+When you are satisfied with your trained model, you can use it to make
+predictions against new data. For example, you can use it in the preprocessor of
+an ingest pipeline or in a pipeline aggregation within a search query. For more
+information about this process, see <<ml-supervised-workflow>> and
+<<ml-inference>>.
+
+You can also supply trained models that are not created by {dfanalytics-job} but
+adhere to the appropriate https://github.com/elastic/ml-json-schemas[JSON schema].
+If you want to use these trained models in the {stack}, you must store them in
+{es} documents by using the {ref}/put-inference.html[create trained model API].
```

Comment:
In the meantime, the URLs are changed and there is a PR up for updating the titles from singular to plural.

```suggestion
{es} documents by using the {ref}/put-trained-models.html[create trained models API].
```